src/main/java/co/novu/api/environments/EnvironmentHandler.java

```diff
@@ -1,44 +1,54 @@
 package co.novu.api.environments;
 
+import java.io.IOException;
+
 import co.novu.api.environments.requests.CreateEnvironmentRequest;
 import co.novu.api.environments.requests.UpdateEnvironmentRequest;
 import co.novu.api.environments.responses.ApiKeyResponse;
 import co.novu.api.environments.responses.BulkEnvironmentResponse;
 import co.novu.api.environments.responses.SingleEnvironmentResponse;
-import co.novu.common.base.NovuConfig;
+import co.novu.common.rest.NovuNetworkException;
 import co.novu.common.rest.RestHandler;
-import lombok.RequiredArgsConstructor;
+import retrofit2.Response;
 
-@RequiredArgsConstructor
 public class EnvironmentHandler {
 
     private final RestHandler restHandler;
 
-    private final NovuConfig novuConfig;
-
-    private static final String ENDPOINT = "environments";
+    private final EnvironmentApi environmentApi;
+    
+    public EnvironmentHandler(RestHandler restHandler) {
```

Comment:
You'd need to modify the constructor of EnvironmentHandler in Novu.java class. Please check that

Reply:
Hi @mayorJAY resolved it